sqlglot/dialects/clickhouse.py

```diff
@@ -21,6 +21,16 @@ def _lower_func(sql: str) -> str:
     return sql[:index].lower() + sql[index:]
 
 
+def _quantile(self, e):
```

Comment:
Nit: I'd add the `_sql` suffix for consistency:

```suggestion
def _quantile_sql(self, e):
```